packages/eslint-config-base/__tests__/linter.js

```diff
@@ -4,19 +4,30 @@ const eslint = require('eslint')
 const RunLinter = require('../../../__tests__/RunLinter')
 const config = require('..')
 
-const { CLIEngine } = eslint
-const eslintRunner = new CLIEngine({
-  ...config,
+const { ESLint } = eslint
+const eslintRunner = new ESLint({
+  overrideConfig: {
+    ...config,
+    rules: {
+      ...config.rules,
+      'no-unused-vars': 'off',
+      'no-constant-condition': 'off',
+      'no-empty': 'off',
+      'class-methods-use-this': 'off',
+      'import/no-unresolved': 'off',
+      'no-unused-expressions': 'off',
+      'import/extensions': 'off',
```

Comment:
Disabled some rules (which are not related with what we are testing) for testing, so it is easier to write code examples, without the need to follow all of them, for example no unused variables or no empty block statements